backend/routes/messages.js

```diff
@@ -12,22 +13,27 @@ const {
 
 const router = express.Router()
 
+// DELETE a message
+router.delete('/:idx', AuthController.roleAuthorization(["admin"]), deleteMessage);
+
 // GET all messages
 router.get('/', AuthController.roleAuthorization(["admin"]), getMessages);
 
+// POST a new message
+router.post('/', createMessage);
+
+// DELETE all messages
+router.delete('/', AuthController.roleAuthorization(["admin"]), deleteMessages);
+
+// GET messages page
+router.get('/messagePages/:userID', AuthController.roleAuthorization(["user", "admin"]), getMessagesPage);
+
 // GET all user's messages
 router.get('/:userID', AuthController.roleAuthorization(["user", "admin"]), getUserMessages)

```

Comment:
Is it Ok? 
Any registered user can get any users messages?

Reply:
I can not understand what the differences between 
'/messagePages/:userID'
and
'/:userID'